cmd/argo/commands/lint.go

```diff
@@ -56,7 +57,9 @@ func NewLintCommand() *cobra.Command {
 					errors.CheckError(err)
```

Comment:
If we attempt to lint a directory, would all the errors be shown as well or just the first one?

Reply:
Will fix this!